test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -555,6 +555,17 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
         $result.Output.Content | Should -BeExactly $expectedResult.Output.Content
     }
 
+    It "Invoke-WebRequest accepts '-ProxyUseDefaultCredentials' parameter" {
+        { Invoke-WebRequest -Uri http://httpbin.org -ProxyUseDefaultCredentials } | Should -Not -Throw
```

Comment:
Don't use off box resources

Reply:
I used current pattern for such tests from the file.